src/chrome/content/rules/dongleauth.info.xml

```diff
@@ -1,9 +1,7 @@
-<!--
-	Invalid Certificate:
-		dongleauth.info
```

Comment:
Please keep this comment that we know the reason for the special redirect below.